### PR TITLE
refactor: define commonly used chunks in a separate file

### PR DESCRIPTION
**Pull request by jorainer**
This PR introduces some rather drastic changes:

- common functions are put into a separate file.
- commonly (repeatedly) used code blocks (chunks) are also put into a separate file.

### CODE REVIEW

File: match-standards-serum-mix01.Rmd [hunk 1]
```diff
@@ -5,44 +5,22 @@ output:
...
-opts_chunk$set(message = FALSE, warning = FALSE, fig.width = 10, fig.height = 8,
-               dev = c("png", "pdf"), fig.path = IMAGE_PATH)
+```{r, include = FALSE, cached = FALSE}
+knitr::read_chunk("R/match-standards-chunks.R")
+MIX <- 1
+MATRIX <- "Serum"
+POLARITY <- "POS"
+```
 
+```{r, libraries, echo = FALSE, message = FALSE}
```
Comment (jorainer) [inline match-standards-serum-mix01.Rmd:21]: this code block will be replaced by `rmarkdown::render` with the code block defined in the *match-standards-chunks.R* file named `## ---- libraries ----`

File: match-standards-serum-mix01.Rmd [hunk 2]
```diff
@@ -5,44 +5,22 @@ output:
...
-MZML_PATH <- "~/mix01/"
-# MZML_PATH <- "/data/massspec/mzML/"
-
-register(SerialParam())
 
-library(knitr)
-opts_chunk$set(message = FALSE, warning = FALSE, fig.width = 10, fig.height = 8,
-               dev = c("png", "pdf"), fig.path = IMAGE_PATH)
+```{r, include = FALSE, cached = FALSE}
+knitr::read_chunk("R/match-standards-chunks.R")
```
Comment (jorainer) [inline match-standards-serum-mix01.Rmd:15]: This reads the definitions of all code chunks from the separate file.

File: match-standards-serum-mix01.Rmd [hunk 3]
```diff
@@ -1195,108 +706,107 @@ The table below lists all features that were matched to one of the adducts of
...
+- MS2 spectra for FT00578, FT00577 and FT00575 match spectra from creatinine.
+- FT00870 (RT=176, `[M+H]+` ion): confidence **C**: the MS2 spectrum F07.S0633
+  matches only `r std` with a low similarity.
+- FT01667 (RT=173.3, `[M+2Ma-H]+` ion): neutral loss spectrum of F07.S0621
+  matches reference spectrum with high simlarity. Confidence **B**?.
+- FT01307 inherit confidence from above.
+- Reference spectra: F07.S0621 (and F07.S0633?).
+
+
+TODO CONTINUE/REPLACE BELOW
```
Comment (jorainer) [inline match-standards-serum-mix01.Rmd:807]: @andreavicini , I've updated all stuff above, please continue here replacing the old code with the new chunks defined above.

**Review by andreavicini — APPROVED** [review]
Looks great to me, thanks @jorainer! I'll merge and work on it

**Pull request merged**